scripts/lib/wic/plugins/source/isoimage-isohybrid.py

```diff
@@ -216,6 +216,18 @@ def do_configure_partition(cls, part, source_params, creator, cr_workdir,
             creator.name = source_params['image_name'].strip()
             logger.debug("The name of the image is: %s", creator.name)
 
+    @staticmethod
+    def _install_payload(source_params, iso_dir):
+        """
+        Copies contents of payload directory (as specified in 'payload_dir' param) into iso_dir
+        """
+
+        if source_params.get('payload_dir'):
+            payload_dir = source_params['payload_dir']
+
+            logger.debug("Payload directory: %s", payload_dir)
+            shutil.copytree("%s" % payload_dir, "%s" % iso_dir, symlinks=True, dirs_exist_ok=True)
```

Comment:
Why are you using string formatting here? Are the params not valid strings already? And, if you want to explicitly typecast them, why not use `str(payload_dir)`?

Reply:
Oh yeah thats unnecessary. Will fix it

Reply:
Done